test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -244,6 +248,35 @@ class CloudFoundryDeployTest extends BasePiperTest {
 
         assertThat(jscr.shell, hasItem(containsString('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"')))
         assertThat(jscr.shell, hasItem(containsString("cf blue-green-deploy testAppName --delete-old-apps -f 'test.yml'")))
+        assertThat(jscr.shell, hasItem(containsString("cf logout")))
+
+    }
+
+    @Test
+    void testCfNativeBlueGreenStopOldInstance() {
```

Comment:
"Stop" and "Keep" are synonyms, right? Please use only one term for the concept.